rustainers/src/runner/inner.rs

```diff
@@ -40,6 +41,14 @@ pub(crate) trait InnerRunner: Display + Debug + Send + Sync {
         Ok(result)
     }
 
+    #[tracing::instrument(level = "debug", skip(self), fields(runner = %self))]
```

Comment:
🤌 **nitpick**: move this function below ``create_network`
I prefer have command grouped by topics